MultiServer.py

```diff
@@ -190,8 +190,43 @@ def __init__(self, host: str, port: int, server_password: str, password: str, lo
         self.stored_data = {}
         self.stored_data_notification_clients = collections.defaultdict(weakref.WeakSet)
 
-    # General networking
+        # init empty to satisfy linter, I suppose
+        self.gamespackage = {}
+        self.item_name_groups = {}
+        self.all_item_and_group_names = {}
+        self.forced_auto_forfeits = collections.defaultdict(lambda: False)
+        self.non_hintable_names = {}
+
+        self.init_datapackage()
+        self._init_datapackage()
```

Comment:
i would try to give those two distinct names. maybe `load_game_data` and `init_game_data` (or `world_data`)? Maybe both with leading `_` to mark them as protected?